src/databricks/labs/remorph/connections/env_getter.py

```diff
@@ -0,0 +1,24 @@
+import os
+import json
+import logging
+
+
+class EnvGetter:
+    def __init__(self, is_debug: bool = False):
```

Comment:
Why not read `debug-env.json `when the app starts and set env variables accordingly (using `os.environ['my_name'] = 'my_value'`) ?

Reply:
That is only for local tests, I don't think it is worth doing that.

Reply:
Then shouldn't we have a DebugEnvGetter class that is passed along, instead of carrying test-only code in prod ?

Reply:
Seperated out.

Reply:
@ericvergnaud ready for review again